src/Diff/Internal/AliasGroupListPatcher.php

```diff
@@ -37,7 +36,7 @@ public function patchAliasGroupList( AliasGroupList $groups, Diff $patch ) {
 	}
 
 	/**
-	 * @see MapPatcher
+	 * @see Diff\Patcher\MapPatcher
```

Comment:
oh, I didn't notice there is no `use Diff\Patcher\MapPatcher` on top of the file (which makes the tool not able to understand the @see thing, which makes sense).